pydantic/main.py

```diff
@@ -31,7 +31,7 @@
 
     AnyClassMethod = classmethod[Any]
     TupleGenerator = typing.Generator[tuple[str, Any], None, None]
-    Model = typing.TypeVar('Model', bound='BaseModel')
+    Model = typing.TypeVar("Model", bound='BaseModel')
```

Comment:
@charliermarsh it looks like this incorrect quote example is not being picked up by ruff (at least for me locally).

Any idea what's going on?

Reply:
Will take a look!

Reply:
Ah, the `Q` codes aren't enabled by default -- only `E` and `F`.

You can change the `pyproject.toml` like this:

```
[tool.ruff]
line-length = 120
select = ["E", "F", "Q"]
flake8-quotes = {inline-quotes = 'single', multiline-quotes = 'double'}
```

Or, equivalently:

```
[tool.ruff]
line-length = 120
extend-select = ["Q"]
flake8-quotes = {inline-quotes = 'single', multiline-quotes = 'double'}
```

Reply:
that worked, thanks. Might be worth adding that to your example of `flake8-quotes`.

Reply:
Good call. (Ruff needs proper docs.)